app/create-event/page.tsx

```diff
@@ -37,10 +37,33 @@ const CreateEvent = () => {
 
     const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
         e.preventDefault();
-        // Save the event in the db through nest.js
         console.log(eventData);
+        createActivity(eventData);
+
     };
 
+    const createActivity = async (activityData: any) => {
+        try {
+            const response = await fetch('http://localhost:3000/api/activity/add', {
```

Comment:
Looks great. Let's also create a ticket that would explore below client libraries (SWR, React Query) to fetch data. 

https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#fetching-data-on-the-client-with-third-party-libraries
https://swr.vercel.app/
https://tanstack.com/query/v3/

Reply:
I included these SWR and React Query links in my google docs task list, which I posted in Slack.